parlai/core/params.py

```diff
@@ -738,6 +738,19 @@ def add_parlai_args(self, args=None):
             choices={None, 'full', 'batchsort'},
             help='Use dynamic batching',
         )
+        parlai.add_argument(
+            '-v',
+            '--verbose',
+            dest='verbose',
+            action='store_true',
+            help='Print all messages',
+        )
+        parlai.add_argument(
```

Comment:
This should actually be just in display_model/interactive, not in generic params.

Reply:
Thanks. will edit that.